syntax = "proto3";
package com.panelsw.ca;
import "errorHandling.proto";
option optimize_for = LITE_RUNTIME;

message ZipDetails{

    bytes zipFile = 1;
    bytes srcFolder = 2;
    bytes includepattern = 3;
    bytes excludepattern = 4;
	bool recursive = 5;	
	ErrorHandling errorHandling = 6;
}
